src/common/guac_rect.c

```diff
@@ -80,3 +80,187 @@ void guac_common_rect_constrain(guac_common_rect* rect, const guac_common_rect*
 
 }
 
+int guac_common_rect_adjust(int divisor, guac_common_rect* rect,
```

Comment:
Yes we could say so. The grid is a floating grid with squares = divisor x divisor